xml/System.Net/WebRequest.xml

```diff
@@ -835,7 +835,7 @@
           <AttributeName>System.ComponentModel.EditorBrowsable(System.ComponentModel.EditorBrowsableState.Never)</AttributeName>
         </Attribute>
         <Attribute>
-          <AttributeName>System.Obsolete("This API supports the .NET Framework infrastructure and is not intended to be used directly from your code.", true)</AttributeName>
```

Comment:
this section here is generated from reflection so we shouldn't touch this part. 
It's what the actual code contains:
![image](https://user-images.githubusercontent.com/12971179/30454430-276a0798-9951-11e7-80cf-04474240574e.png)